src/TideEntityUpdateHelper.php

```diff
@@ -608,4 +620,67 @@ public function import($type, $name, $prioritise_sync = TRUE) {
     return TRUE;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function revert($type, $name, $prioritise_sync = TRUE) {
```

Comment:
Could you explain a little bit what this is trying to do? I don't get why a revert function would `$prioritise_sync`?

Reply:
Does it try and fill in missing values from install config based on what is already there?

Reply:
By default, when `$prioritise_sync` is `true`, the `revert` function will first check  `/app/config/sync`. If the file isn't found there, it will look in the module’s `config/install`, and then in `config/optional`.

If `$prioritise_sync` is set to `false`, the function skips `/app/config/sync` and checks only `config/install`, then `config/optional`.

This makes it easier for developers to **update** specific config entities, for example, custom module defined YAML files in content-vic. The `revert` function handles UUIDs internally, so it won't cause any UUID changes.